flex-config/generic.lua

```diff
@@ -255,25 +257,25 @@ function osm2pgsql.process_relation(object)
     end
 
     if relation_type == 'route' then
-        tables.routes:add_row({
+        tables.routes:insert({
             tags = object.tags,
-            geom = { create = 'line' }
+            geom = object:as_multilinestring()
         })
         return
     end
 
     if relation_type == 'boundary' or (relation_type == 'multipolygon' and object.tags.boundary) then
-        tables.boundaries:add_row({
+        tables.boundaries:insert({
             tags = object.tags,
-            geom = { create = 'line' }
+            geom = object:as_multilinestring()
```

Comment:
Shouldn't there be a line_merge to have the same result as before?

Reply:
fixed in new version